diagnostic.md

```diff
@@ -5,74 +5,86 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The back end is the the database or model in MVC terminology. Without it, your
+would not be able to save or commit any data to the back end data store or base. Think of Amazon.com that is not attached to a datbase. Not possible to function.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model is reponsible for fetching the data. A Model directly manages the data in our application, and provides a representation of that data for the rest of the application to use.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Only the controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+In API-style applications, serializers replace all the views that we care about so we can safely skip the views as well.
```

Comment:
Close, because we are building SPAs, we handle the view states on the front-end to reduce the calls we make to the back-end.